### PR TITLE
COVID-19 Education Modules Doc

### PR DESCRIPTION
**Pull request by sabinasobhani**
Added a new page in Medic's documentation to cover the work that a group of volunteers has been doing on a reference application for education modules surrounding COVID-19.

### CODE REVIEW

File: content/en/apps/examples/training/covid-training.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "COVID-19 Digital Education"
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:2]:
```suggestion
title: "COVID-19 CHW and Patient Education"
```

File: content/en/apps/examples/training/covid-training.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+linkTitle: "COVID-19 Training"
+weight: 
+description: >
+  App and SMS training for CHWs to combat COVID-19.
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:6]:
```suggestion
 Training for CHWs to combat COVID-19.
```

File: content/en/apps/examples/training/covid-training.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+keywords: training onboarding 
+---
+
+Three learning modules were created, both via the CHT and via SMS and messaging-based applications, to rapidly and remotely train CHWs on COVID-19. The combination of CHT and SMS enables Supervisors to train CHWs on workflows without being physically present, and cater to those without reliable data connection through SMS. Each learning module contains a quiz at the end to reinforce learning, which can be tracked by Supervisors. The application covers the below three modules:
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:10]:
```suggestion
The COVID-19 pandemic has created unique challenges to providing in-person CHW training. To support CHWs, three learning modules were created to rapidly and remotely train them on COVID-19 safety protocols and patient care. Through CHT app and SMS deployments, Supervisors can now train CHWs on care workflows without being physically present. This example covers the following learning modules:
```

File: content/en/apps/examples/training/covid-training.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+* Health safety protocols for preventing the spread of COVID-19
+* Preventing the dissemination of misinformation about COVID-19
+* Recognizing COVID-19 and caring for patients with suspected infection
+
+## Problem Being Addressed
+
+Now, more than ever, it is important for health care workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. This training would traditionally be done in person, however health systems must adapt and still deliver this content in a timely and effective manner. A form-based application was designed using the CHT to deliver three learning modules, utilizing tasks to order the module and enable the CHW to progress through.
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:18]:
```suggestion
Now, more than ever, it is important for community health workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. This training would traditionally be done in person, however, due to public health concerns, travel and gatherings of people are limited. Health systems have been forced to quickly adapt to deliver this training content in a safely distanced, timely, and effective manner.  
```

File: content/en/apps/examples/training/covid-training.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+* Health safety protocols for preventing the spread of COVID-19
+* Preventing the dissemination of misinformation about COVID-19
+* Recognizing COVID-19 and caring for patients with suspected infection
+
+## Problem Being Addressed
+
+Now, more than ever, it is important for health care workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. This training would traditionally be done in person, however health systems must adapt and still deliver this content in a timely and effective manner. A form-based application was designed using the CHT to deliver three learning modules, utilizing tasks to order the module and enable the CHW to progress through. 
+Given COVID-19 is impacting even the hardest-to-reach communities, the modules can also be delivered via SMS to support CHWs who either don't have access to a smartphone or the CHT app, or don't have access to reliable data connection.
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:19]:
```suggestion
COVID-19's impact on the hardest-to-reach communities that often lack access to a reliable data connection, poses additional challenges to reaching the CHWs that provide care to people at the highest health risk. 
```

File: content/en/apps/examples/training/covid-training.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+* Recognizing COVID-19 and caring for patients with suspected infection
+
+## Problem Being Addressed
+
+Now, more than ever, it is important for health care workers to stay safe and serve their communities by understanding effective health protocols for preventing the spread of COVID-19. This training would traditionally be done in person, however health systems must adapt and still deliver this content in a timely and effective manner. A form-based application was designed using the CHT to deliver three learning modules, utilizing tasks to order the module and enable the CHW to progress through. 
+Given COVID-19 is impacting even the hardest-to-reach communities, the modules can also be delivered via SMS to support CHWs who either don't have access to a smartphone or the CHT app, or don't have access to reliable data connection. 
+
+## Solution Overview
+
+The CHT application, combined with SMS workflow, offers a flexible way to deliver timely COVID-19 information to CHWs. The three learning modules:
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:23]:
```suggestion
The CHT provides a flexible way to deliver timely COVID-19 remote training and education to CHWs. In collaboration with [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/), and vetted by a team of health experts at Stanford, a form-based application was designed to deliver three remote learning modules. Utilizing tasks to order the modules, CHWs can progress through training at their own pace. The training modules:
```

File: content/en/apps/examples/training/covid-training.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+* Build on verifiable health safety guidance from the CDC and WHO
+* Were carefully crafted by [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/) team and vetted by a team of health experts at Stanford
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:26]: pulled this into the solution body, so can be deleted

File: content/en/apps/examples/training/covid-training.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+* Can be deployed using the CHT app or SMS workflows 
+* Capture assessment and understanding of knowledge through quizzes
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:28]:
```suggestion
* Capture assessment and understanding of CHW knowledge through quizzes, which can be tracked by Supervisors
```

File: content/en/apps/examples/training/covid-training.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+### Remote Training by SMS
+
+The workflow involves interacting with the CHW through SMS and utilizing text-based feedback from the CHW, along with branching logic, to promote engagement and absorption of content. CHWs can be prompted with a question and reply via a message to assess understanding of content.
+
+![workflow.png](workflow1.png)
+
+## Download Additional Content
+
+For more detailed information and to download detailed content, please reference the documentation for both the [CHT application](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg/edit) and the [SMS and Messaging workflow](https://docs.google.com/document/d/1fPqZhl5gYoW_UJx6E3EWngWcICOH8mfmslo4r5JqmE0/edit?usp=sharing). These showcase the content, key considerations for adapting the content, and the technical implementation.
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:54]: might want to change these docs access to "view only" before they are published live

File: content/en/apps/examples/training/covid-training.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+![hierarchy.png](hierarchy.png)
+
+## Workflow Examples
+
+### Remote Training by App
+
+This demo illustrates how a CHT workflow can be easily adapted to integrate a sequence of learning modules through the usage of tasks. These modules can be delivered sequentially and learning can be tracked across CHWs. 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/pFEFIY_SA7M" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:44]:
```suggestion
{{< youtube pFEFIY_SA7M >}}
```
updated to match style guide

File: content/en/apps/examples/training/covid-training.md (hunk 9 above)
Comment (MaxDiz) [inline content/en/apps/examples/training/covid-training.md:54]:
```suggestion
For more detailed information and to download detailed content, please reference the documentation for both the [CHT application](https://docs.google.com/document/d/1Vd6x_WwS-S_sekRQnvW1UAS_rWaTnaawITfeXgX6Sdg) and the [SMS and Messaging workflow](https://docs.google.com/document/d/1fPqZhl5gYoW_UJx6E3EWngWcICOH8mfmslo4r5JqmE0). These showcase the content, key considerations for adapting the content, and the technical implementation. 
```

**Comment by garethbowen** [comment]
@MaxDiz Would you mind polishing this up and getting it ready to merge?

**Comment by MaxDiz** [comment]
@abbyad This example was drafted a while ago, but lost in the shuffle. I cleaned it up to match the other examples. We have a Community event on Covid-19 next week (Nov 16), so it would be nice to have this up on the site before then.

It's showing a conflict with the "Remote Training" page. I think this is because it is a sub-page currently (but not sure). It may make more sense to have this page on the same hierarchy level as the others. What do you think?

**Comment by abbyad** [comment]
Ya, I agree that this new page could be at the `examples` level, and doesn't need to be nested in Training. In doing so you can exclude moving the existing training page, so you won't have a conflict.

**Comment by MaxDiz** [comment]
@abbyad I moved to the main level within the "examples" hierarchy. Please approve if you think it's good-to-go

**Comment by abbyad** [comment]
@MaxDiz I don't see any changes since my last comment. Are they in this branch? If so, is it possible they haven't been pushed?

**Comment by abbyad** [comment]
Closing as per https://github.com/medic/cht-docs/pull/401:

> covid education and training module (replaces #269 which has a conflict I cannot resolve)

**Pull request closed without merging**